ReactWindows/ReactNative/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -527,6 +525,19 @@ private void OnKeyDown(object sender, KeyRoutedEventArgs e)
                             textBox.GetTag(),
                             textBox.Password));
             }
+
+            if (!e.Handled)
+            {
+                var keyCode = e.Key.GetKeyCode();
+                textBox.GetReactContext()
+                    .GetNativeModule<UIManagerModule>()
+                    .EventDispatcher
+                    .DispatchEvent(
+                        new KeyEvent(
+                            KeyEvent.KeyDownEventString,
```

Comment:
>KeyDownEventString [](start = 37, length = 18)

Why does this control fire key down events but not key up events?

Reply:
I could add keyup for symmetry. It was just keydown used all over.

Reply:
More, I'll add this to prop types as well, currently we just use the bubbling aspect, but adding proptypes would make it more complete.

Reply:
Fixed (see next comment)

---
In reply to: [156585637](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156585637) [](ancestors = 156585637)